Dan Tudor/05-3.5/main.c

```diff
@@ -0,0 +1,110 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+void addition(int **K,int **B,int **C,int rows, int cols)
+{
+    int i,j;
+    for(i=0; i<rows; i++)
+    {
+        for(j=0; j<cols; j++)
+        {
+            K[i][j] = B[i][j] + C[i][j];
+        }
+    }
+}
+
+void subtraction(int **A, int **P, int **K, int rows, int cols)
+{
+    int i,j;
+    for(i=0; i<rows; i++)
+    {
+        for(j=0; j<cols; j++)
+        {
+            A[i][j] = P[i][j]-K[i][j];
+        }
+    }
+}
+
+void product(int **P, int **B, int **C, int rows, int cols)
+{
+    int i,j,k,s;
+    for(i=0; i<rows; i++)
+    {
+        for(j=0; j<cols; j++)
+        {
+            s=0;
+            for(k=0; k<rows; k++)
+            {
+                s = s+B[i][k]*C[k][j];
+            }
+            P[i][j] = s;
+        }
+
+    }
+}
+
+int main()
```

Comment:
Nice and clean!

Remind me to tell you guys how you can extract the number of rows and columns from a matrix such that you don't need to carry those values as parameters to all functions